syntax = "proto3";
package Proto;

import "Exchange.proto";
import "Request.proto";
import "Reply.proto";

message Destriker
{
  string instrument = 1;
  double destriker = 2;
}

message DestrikerReq
{
  RequestType type = 1;
  Exchange exchange = 2;
  repeated Destriker destrikers = 3;
  string instrument = 4;
  string user = 5;
}

message DestrikerRep
{
  repeated Destriker destrikers = 1;
  Reply result = 2;
}
